docs/section10/10-4.md

```diff
@@ -0,0 +1,58 @@
+# ASHVAC – Rule 10-4  
+
+**Rule ID:** 10-4  
+ 
+**Rule Description:** Baseline HVAC systems using fossil fuel shall be modeled using natural gas. Except where natural gas is not available for the proposed building site, propane shall be used as the heating fuel.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building   
+**Appendix G Section Reference:**   
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**   
+
+1. Applicable to baseline air-side systems 3 and 9.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_heating_fuels_p()
+
+
+**Manual Check:** None   
+ 
+
+**Rule Logic:**  
+**Applicability Check 1:**   
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get list of P_RMR heating_fuels: `heating_fuels_p = get_heating_fuels_p(P_RMR)`
+- Check if oil, other or propane are in the P_RMR: `if "PROPANE" in heating_fuels_p or "FUEL_OIL" in heating_fuels_p or "OTHER" in heating_fuels_p: oil_prop_other_p = TRUE`
+- Else: `oil_prop_other_p = FALSE`
+- For each hvac system in the B_RMR: `for hvac_b in B_RMR...HeatingVentilationAirConditioningSystem:`
+    - Get hvac system type for hvac_b: `hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_b.id)]`
+    - Check if HVAC system is type 3 or 9: `if hvac_sys_type_b in ["SYS-3", "SYS-3a", "SYS-9"]:`
```

Comment:
What about systems 1, 5, 7 and 12? But as I mentioned above, I would check fuels for all heating sources irrespective of system type.

Reply:
These are served by boilers (i.e. HW coils) so I left them out. I thought the boiler fuel is being checked with the waterside checks. If not then I will need to check boilers as well. Discuss on call.